EIPS/eip-7160.md

```diff
@@ -0,0 +1,163 @@
+---
+eip: 7160
+title: ERC-721 Multi-Metadata Extension
+description: "This EIP proposes an extension to the ERC-721 to support multiple metadata URIs per token."
+author: 0xG (@0xGh), Marco Peyfuss (@mpeyfuss)
+discussions-to: https://ethereum-magicians.org/t/erc721-multi-metadata-extension/14629
+status: Draft
+type: Standards Track
+category: ERC
+created: 2023-06-09
+requires: 165, 721
+---
+
+## Abstract
+
+This EIP proposes an extension to the [ERC-721](./eip-721.md) standard to support multiple metadata URIs per token. It introduces a new interface, `IERC721MultiMetadata`, which provides methods for accessing the metadata URIs associated with a token, including a pinned URI index and a list of all metadata URIs. The extension is designed to be backward compatible with existing `ERC721Metadata` implementations.
+
+## Motivation
+
+The current [ERC-721](./eip-721.md) standard allows for a single metadata URI per token. However, there are use cases where multiple metadata URIs are desirable, such as when a token represents a collection of (cycling) assets with individual metadata, historic token metadata, collaborative and multi-artist tokens, evolving tokens. This extension enables such use cases by introducing the concept of multi-metadata support.
```

Comment:
I think it would be important to include answers to:

 - Why does this need to be more than just `tokenURI` and some token-specific functions to change the active URI?
 - Who would be interested in the non-pinned URIs?
 - How would this information be displayed?

Reply:
Added a paragraph right below. Let me know what you think. Also @mpeyfuss can chime in and add further reasons for why having a multi-metadata standard might be a good idea.

Reply:
Yeah there are quite a few examples we've found working with creators and collectors on why multi-metadata is interesting. Can add example use cases. One of the most mentioned reasons is that of having the option to provide versions of the token that match different aspect ratios and could be utilized for specific display purposes while keeping provenance/ownership intact.